dsp/modules/__init__.py

```diff
@@ -6,6 +6,7 @@
 from .cohere import *
 from .sbert import *
 from .pyserini import *
+from .llama import *
```

Comment:
Should we remove this line?

Reply:
Alternatively, we can move `from llama_cpp import Llama` into _inside_ `class LlamaCpp(LM)`.

What do you prefer?

Reply:
What would be the benefit of this? In the current implementation I think it follows the same structure as cohere